test/jest/victory-native/native-helpers.test.js

```diff
@@ -1,4 +1,4 @@
-import NativeHelpers from "../../packages/victory-native/src/helpers/native-helpers";
+import NativeHelpers from "../../../packages/victory-native/src/helpers/native-helpers";
```

Comment:
Note that this should already work without any prefix (and make integration with my branch easier) as:

```suggestion
import NativeHelpers from "victory-native/src/helpers/native-helpers";
```